core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -993,6 +999,18 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleApiVersionRequest(request: RequestChannel.Request) {
+    val responseHeader = new ResponseHeader(request.header.correlationId)
+    def isApiVersionRequestVersionSupported: Boolean = {
```

Comment:
Similar to the above comment on the `UnknownApiVersionRequestVersionException`, [KAFKA-2512](https://issues.apache.org/jira/browse/KAFKA-2512) adds a check for all of the apis and therefore should handle this api too. See the pull request for details: https://github.com/apache/kafka/pull/200 cc @becketqin